src/main/java/com/example/turboaz/controllers/ListingController.java

```diff
@@ -23,6 +21,8 @@
     ListingService service;
     ImageService imageService;
     TransactionService transactionService;
+    @Autowired
+    EmailService emailService;
 
```

Comment:
If you make it Autowired better to make EmailService private

Reply:
absolutely